### PR TITLE
Update zh_TW PO and MO

### PR DESCRIPTION
**Pull request by alexclassroom**
Hi Andy,

I reviewed the translations I provided and found some of them were not so good.

This PR means all modified translations existed my project forked from yours, I think you can merge them directly.

Regards,
Alex

### CODE REVIEW

**Comment by pedro-mendonca** [comment]
@alexclassroom why are you deleting all the language files?

**Comment by afragen** [comment]
@alexclassroom please change only the files where you have made changes. If you're having problems figuring out how to make a PR we can go back to you providing the changed files for me to download.

Merging this would delete all the language files except Chinese. That's probably not what you intended.

**Comment by afragen** [comment]
Merging this would delete all the language files except Chinese.

File: languages/github-updater-zh_TW.po [hunk 1]
```diff
@@ -328,25 +328,28 @@ msgstr "Bitbucket Server 私有版本庫"
...
-"延伸命名重新命名外掛目錄 %s，以防止與 WordPress.org 上的外掛發生衝突。"
+"[延伸命名] 會將外掛目錄重新命名為 %s，以防止與 WordPress.org 上的外掛發生衝"
+"突。"
 
 #: src/GitHub_Updater/Settings.php:761
 #, php-format
 msgid "Activate Extended Naming by setting %s"
-msgstr "在程式碼內加入 %s 以啟用延伸命名。"
+msgstr ""
+"在 wp-config.php 或你使用的佈景主題的 functions.php 內加入 %s 以啟用 [延伸命"
```
Comment (afragen) [inline languages/github-updater-zh_TW.po]: @alexclassroom I purposely do not go into detail about how to do this.

**Comment by afragen** [comment]
Hey Alex. It looks like that last update to your PR was exactly opposite of what we needed. If it's easier to have me download the new files we can do that.

**Comment by alexclassroom** [comment]
Hi Alan,

I am sorry about this because I am still learning to use Github.

I reverted my branch, now all other languages' PO and MO files are back and modified Traditional Chinese files are there.

If I still did wrong operation for this, files' download link is https://app.box.com/s/6qc9djub5plkg3oj660dvfus19qvumu5, it should be same as previous link yesterday.

Thanks for your patience.

Regards,

Alex

**Comment by afragen** [comment]
I pulled the files from the .box location

**Pull request closed without merging**